drivers/sensor/paw3212/paw3212.c

```diff
@@ -285,7 +286,21 @@ static void handler_trigger(struct k_work *work)
 		.chan = SENSOR_CHAN_ALL,
 	};
 
-	handler(DEVICE_GET(paw3212), &trig);
+	if (handler) {
+		handler(DEVICE_GET(paw3212), &trig);
+	}
```

Comment:
If `handler` is `NULL` at this point, it means that the trigger was disabled somewhere between IRQ handler submitted the work and the work got to be executed, so there is no need to perform further operations here (even in case the trigger was re-enabled right before `if (handler)`, its callback will be already active, there's no need to attempt to enable it again).

Reply:
Fixed